.github/workflows/build-and-test.yml

```
@@ -0,0 +1,32 @@
+        uses: actions/setup-dotnet@v4
+        with:
+          dotnet-version: '8.0.x'
+
+      - name: Restore dependencies
+        run: dotnet restore
+
+      - name: Build solution
+        run: dotnet build --configuration Release --no-restore
+
+      - name: Run tests
+        run: dotnet test --no-restore --verbosity normal
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)